src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -94,15 +91,14 @@
     </ul>
     <button on:click={logout}>Logout</button>
   {:else}
-    <button
-      on:click={() =>
-        modal.open({
+    <button on:click={() => {
+        journeyEvents?.start({
```

Comment:
So what happens if multiple different "journeyEvents" variables are scattered throughout an app? I assume because its an observable even if i have say 5 files creating their own "subscriptions" throughout a lifecycle of an app, they are all subscribing to the same source of truth so therefore we don't have to worry about them getting out of sync? Does that make sense / is my logic here correct?

Reply:
Yes, they would each have their own subscription to the same underlying stores, so their no syncing issue that I can think of. The only thing we would want to warn against is not littering your codebase of these subscriptions without cleaning them up. This would lead to a memory leak. This is why we should encourage devs to place these at higher level components of their app or within their central state management solutions.

One way to avoid this if they really want to do a lot of one-off throughout their app is to use the Promise-based pattern, instead of the Observable-based pattern. The consequence is they just get the single, final value, but the good is we internally subscribe then unsubscribe for them, so no memory leaks.

Reply:
I think this needs to be clearly signposted - are we going to update readme / docs in this PR with warnings or advice, or in a separate one?

Reply:
@daveadams56 Absolutely! We will have to rework our documentation to repeatedly ensure we lead them to the right decisions. Documentation will have to be a quick follow up to this story/PR.